docs/developer-docs/latest/development/export-import.md

```diff
@@ -0,0 +1,256 @@
+
+- the project configuration,
+- entities,
+- links,
+- assets,
+- schemas,
+- `metadata.json` file.
+
+### Name the export file
+
+Exported data are contained in a `.tar` file that is automatically named using the format `export_time-date-stamp`. You can optionally name the exported file by passing the `--file` or `-f` option with the `strapi export` command. Do not include a file extension.
+
+#### Example: Export data with a custom filename
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --file my-strapi-export
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export  --file my-strapi-export
+```
+
+</code-block>
+</code-group>
+
+
+### Configure data encryption
+
+The default `strapi export` command encrypts your project data using `aes-128-ecb` encryption. To use encryption you need to pass an encryption key using the `-k` or `--key` option or enter an encryption key when prompted. The encryption key is a `string` with no minimum character count.
+
+To disable encryption, pass the `--no-encrypt` option with the `strapi export` command.
+
+#### Example: Export data without encryption
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --no-encrypt
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --no-encrypt
+```
+
+</code-block>
+</code-group>
+
+#### Example: Export data with the encryption `--key` option
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --key my-encryption-key
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --key my-encryption-key
+```
+
+</code-block>
+</code-group>
+
+### Disable data compression
+
+The default `strapi export` command compresses your project data using `gzip` compression.
+
+To disable compression, pass the `--no-compress` option with the `strapi export` command.
+
+#### Example: Export data without compression
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --no-compress
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --no-compress
+```
+
+</code-block>
+</code-group>
+
+<!-- ### Declare the maximum file size
+
+The default maximum size of each internal backup `jsonl` file is 256MB. To customize the maximum `jsonl` file size, pass the `--max-size-jsonl` option with the `strapi export` command.
+
+#### Example: Set the maximum `jsonl` file size to 100MB
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi export --max-size-jsonl 100
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi export --max-size-jsonl 100
+```
+
+</code-block>
+</code-group> -->
+
+<!-- ### Exclude files -->
+
+## Import data using the CLI tool
+
+To import data into a Strapi instance use the `strapi import` command in the project root directory. Specify the file to be imported using the `-f` or `--file` option. The filename, extension, and path are required.
```

Comment:
I think it's critical here to mention that we've currently only implemented a "restore" that will wipe all the data from the existing Strapi project